build.gradle

```diff
@@ -1,8 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.0'
-    ext.agp_version = '3.3.0'
+    ext.kotlin_version = '1.3.31'
+    ext.agp_version = '3.4.1'
```

Comment:
Updated the Android PlugIn to get access to the new method `ViewDataBinding.getLifecycleOwner()`
This required updating the Kotlin Plugin as well.